src/components/application_manager/src/resumption/resume_ctrl_impl.cc

```diff
@@ -324,8 +336,11 @@ void ResumeCtrlImpl::OnSuspend() {
 
 void ResumeCtrlImpl::OnIgnitionOff() {
   LOG4CXX_AUTO_TRACE(logger_);
-  resumption_storage_->IncrementIgnOffCount();
-  FinalPersistData();
+  if (!application_manager_.IsLowVoltage()) {
+    resumption_storage_->IncrementIgnOffCount();
```

Comment:
still need `IncrementIgnOffCount()`?

Reply:
@yang1070 This is different counter. Not for low Voltage handling. It existed before and used for different purposes.